src/drivers/px4fmu/fmu.cpp

```diff
@@ -1026,74 +1026,75 @@ PX4FMU::cycle()
 			}
 		}
 
-		/* can we mix? */
-		if (_mixers != nullptr) {
+	}
 
-			size_t num_outputs;
+	_cycle_timestamp = hrt_absolute_time();
```

Comment:
Then I guess my FMU bugfix is correct, since it guarantees that actuator_outputs are published on every cycle.